src/core/utils.js

```diff
@@ -379,19 +380,29 @@ export class IDBKeyVal {
 // Takes an array and returns a string that separates each of its items with the proper commas and
 // "and". The second argument is a mapping function that can convert the items before they are
 // joined
-export function joinAnd(array = [], mapper = item => item) {
+export function joinAnd(array = [], mapper = item => item, lang = docLang) {
   const items = array.map(mapper);
-  switch (items.length) {
-    case 0:
-    case 1: // "x"
-      return items.toString();
-    case 2: // x and y
-      return items.join(" and ");
-    default: {
-      // x, y, and z
-      const str = items.join(", ");
-      const lastComma = str.lastIndexOf(",");
-      return `${str.substr(0, lastComma + 1)} and ${str.slice(lastComma + 2)}`;
+  if (Intl.ListFormat && typeof Intl.ListFormat === "function") {
+    const formatter = new Intl.ListFormat(lang, {
+      style: "long",
+      type: "conjunction",
+    });
+    return formatter.format(items);
+  } else {
```

Comment:
Nit: you don’t else here, as the code returns above.

Reply:
I must say, I saw that coming :laughing: